### ISSUE TITLE
1.3.0 on C* 1.2.19: cannot load cluster state (`nodes_status` remains null)

### ISSUE THREAD

**Issue by rborer** [thelastpickle/cassandra-reaper#579]
Hi,

I recently upgraded reaper from 1.2.2 to 1.3.0 and since then the UI is not able to load the cluster state anymore, it keeps spinning (it doesn't show neither the disk usage neither the nodes):

![spinning](https://user-images.githubusercontent.com/361403/47523062-70448480-d897-11e8-9d26-6f34212562a0.png)

Looking at the ajax calls made I can see that the endpoint `cluster/staging-cluster` returns a json object with `nodes_status` set to `null`:

![image](https://user-images.githubusercontent.com/361403/47523298-02e52380-d898-11e8-9ddc-72b507b69487.png)

In `cassandra-reaper.log` I can see that the following exception happens frequently:

```
Caused by: javax.management.InstanceNotFoundException: org.apache.cassandra.net:type=StreamManager
```

Note that I haven't checked the code (yet) so I don't know if this may be related or not.

Does it means that reaper 1.3.0 is not compatible with Cassandra 1.2 anymore?

Cheers,
Reynald

**Comment by rborer** [comment]
I can confirm that on my 1.2.19 cluster #580 restores the `nodes_status` entry:

![image](https://user-images.githubusercontent.com/361403/47549606-da434500-d8fc-11e8-9cb3-7bd7e9b6b727.png)

Also, the `/streams/` endpoint returns an empty list of streams (since this is not supported).

**Comment by rborer** [comment]
Fixed by #580.
